src/sequelize.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 import isPlainObject from 'lodash/isPlainObject';
+import { initDecoratedModel } from './decorators/shared/model';
```

Comment:
Don't think this is used, right?

Reply:
can't wait until the whole codebase is written in TS so our linter catches this

Reply:
Done